packages/client/.eslintrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   extends: '../../config/eslint.js',
   parserOptions: {
+    extraFileExtensions: ['.json'],
```

Comment:
Does this include/activate linting on JSON files?

Reply:
Ah no, I think linting for JSON is great. 👍 (not sure if there are other opinions here)